### PR TITLE
Redirect to the Defacto websites

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Redirect all pages to the ones of CAPP Agile Learning on the Defacto websites. I couldn't get the [Middleman redirects](https://middlemanapp.com/basics/redirects/) to work so I decided to put it in layout.erb.
- [x] Build ignore everything that's not a page by renaming the folders and files with a leading _

### CODE REVIEW

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -1,27 +1,11 @@
...
-    <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <title><%= page_title %></title>
-    <meta name="description" content="<%= page_description %>">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <%= favicon_tag "favicon.ico" %>
-    <%= stylesheet_link_tag "site" %>
-    <%= partial "og_tags" %>
-    <%= partial "google_analytics" unless config[:mode] == :server -%>
-    <%= javascript_include_tag "//ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js", :all %>
+    <link rel="canonical" href="<%= t("urls.redirect") %>" />
```
Comment (sn3p) [inline source/layouts/layout.erb]: I would not add the trailing `/`, which is basically xhtml.

**Comment by dkhgh** [comment]
What matthijs said

**Comment by RonaldDijkstra** [comment]
1. No, we need that pages to build for redirects
2. Referrer
3. Ok!

**Comment by RonaldDijkstra** [comment]
I made a branch **original** and protected it from deletion.

**Pull request merged**